### DISCUSSION TITLE
Support for Automatic Class Sorting with Prettier

### DISCUSSION THREAD

**Discussion by andrewdoro** [heroui-inc/tailwind-variants#19] [Q&A] [answered]
Prettier has this plugin for automatic class sorting. Currently it's not working with TW. I am not sure how to make it work. It might require some changes to the plugin to support regex.
https://tailwindcss.com/blog/automatic-class-sorting-with-prettier

**Comment by hauptrolle** [comment]
I would like to have that feature as well. 👍

**Comment by hauptrolle** [comment]
Any updates on this one @jrgarciadev ?

**Comment by zwgnr** [comment, marked as answer]
Not sure if this is a stale issue and you guys have this sorted. If not, I have a working solution.

In your prettier config add  tailwindFunctions:['tv']

The below image is from an Astro project, but should work in any framework. 

![image](https://github.com/nextui-org/tailwind-variants/assets/86675944/b533285a-683b-4c3e-b31f-b1023920b0b4)

**Reply by Blankeos** [reply]
Thanks for this! Is this written in the documentation?

**Reply by zwgnr** [reply]
You're welcome! I don't think it is mentioned anywhere in the docs currently. Seems like it would be a helpful addition though.

**Reply by mskelton** [reply]
This would be a great addition to the docs, thanks for the info!

**Reply by mskelton** [reply]
PR opened to document this: https://github.com/nextui-org/tailwind-variants-docs/pull/20
